upup/models/cloudup/resources/addons/cluster-autoscaler.addons.k8s.io/k8s-1.15.yaml.template

```diff
@@ -134,6 +134,12 @@ spec:
     metadata:
       labels:
         app: cluster-autoscaler
+      {{ if WithDefaultBool .Metrics false }}
```

Comment:
If the only thing this flag does is add the annotation, it is not that useful. You could just add the annotations without the flag. Typically such a flag would be used for enabling/disabling CAS serving metrics in the first place.

Reply:
removed flag also ^^